CHANGELOG.md

```diff
@@ -10,6 +10,7 @@
 #### Fixes
 
 * Your contribution here.
+* [#2033](https://github.com/ruby-grape/grape/pull/2033): Ensure floats are correctly coerced to bigdecimal - [@tlconnor](https://github.com/tlconnor).
```

Comment:
quote and spell `Float` and `BigDecimal`

Reply:
Fixed.